Strategy11/formidable-forms
# Make conditional logic dropdown options label/value update with field options

Fix https://github.com/Strategy11/formidable-pro/issues/2911
Requires https://github.com/Strategy11/formidable-pro/pull/4940

### Test steps

1. Create a new form with a radio button field. Check 'Use separate values'.
2. Add a conditional field based off of the radio.
3. Update the the radio options to something else, including the option used in the conditional logic in (2).
4. The conditional logic value options should be updated to reflect the change.
5. Confirm that the field labels and values are mapped to the conditional logic dropdown correctly
